README.md

```diff
@@ -1,3 +1,5 @@
+***Note - This repo has been archived as of Tuesday, January 7th. The DNS-over-HTTPS extension [has been migrated](https://searchfox.org/mozilla-release/source/browser/extensions/doh-rollout) to the Firefox browser as of [Release 72](https://www.mozilla.org/en-US/firefox/72.0/releasenotes/). Please file any issues regarding this extension through [Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi).***
```

Comment:
Suggest adding the year 2020 to the date. Otherwise LGTM

Reply:
+1 for adding 2020 to the date.
Also, please use this URL for the Bugzilla link: https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&blocked=1598218&product=Firefox&component=Security
It populates the product/component and sets the bug as blocking the bug that moved the code.

Reply:
Update both notes. Thanks, all!